MVP Module/Source/Feed Presentation/FeedErrorViewModel.swift

```
@@ -0,0 +1,13 @@
+//
+//  FeedErrorViewModel.swift
+//  MVP
+//
+//  Created by Danil Vassyakin on 11/4/20.
+//  Copyright © 2020 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+struct FeedErrorViewModel {
+    let errorMessge: String
```

The `errorMessage` could be an optional `String?`, so the Presenter can decide when to show/hide the message.

For example, in the `didStartLoadingFeed()` method, the Presenter can call  `feedErrorView.display(.none)` to hide the error.